$NetBSD: patch-src_bc_configure.ac,v 1.3 2024/01/05 14:55:39 riastradh Exp $

Allow CONFIGURE_ENV to set PAXCTL so user need not have /usr/sbin in
PATH.

--- src/bc/configure.ac.orig	2023-11-20 06:06:53.000000000 +0000
+++ src/bc/configure.ac
@@ -548,7 +548,7 @@ case "$host_os" in
     LIBS="$LIBS -rdynamic"
     skip_iconv_check=yes
     add_curses_lib="-lterminfo"
-    default_post_linker="paxctl +m"
+    default_post_linker="${PAXCTL:-paxctl} +m"
     ;;
   irix*)
     enable_cgcdefault="yes"
